egasub/ega/entities/experiment.py

```diff
@@ -4,7 +4,7 @@ class Experiment(object):
     def __init__(self, alias, title, instrument_model_id, library_source_id, library_selection_id,
                  library_strategy_id, design_description, library_name, library_construction_protocol,
                  library_layout_id, paired_nomial_length, paired_nominal_sdev,
-                 sample_id, study_id,id, status=None):
+                 sample_id, study_id,_id, status=None):
```

Comment:
I think the recommendation is to use trailing underscore to avoid conflicts with Python keyword

Check here: https://www.python.org/dev/peps/pep-0008/, and search for `single_trailing_underscore_`